src/components/PriceBar/index.tsx

```diff
@@ -43,6 +45,9 @@ const PriceBar: React.FC = () => {
   return (
     <>
       <Container>
+        <OpenCloseButton>
+          <AiOutlineDown />
+        </OpenCloseButton>
```

Comment:
Is this for card cc007 or cc008? If for cc008, the button should open and close the price bar as I read from our notion board.
The solution can be applied by using a state conditionally render the price bar.

Reply:
This is for cd007 AC1, "I can see the price bar with “v” on the top." Just for UI purpose.
The open close button feature which open/close the price bar will be implemented in cc008.